edgelet/iotedge/src/init/import/mod.rs

```diff
@@ -0,0 +1,701 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+mod old_config;
+
+use std::path::Path;
+
+use config::Config;
+
+use edgelet_utils::YamlFileSource;
+
+const AZIOT_KEYD_HOMEDIR_PATH: &str = "/var/lib/aziot/keyd";
+const AZIOT_CERTD_HOMEDIR_PATH: &str = "/var/lib/aziot/certd";
+const AZIOT_IDENTITYD_HOMEDIR_PATH: &str = "/var/lib/aziot/identityd";
+const AZIOT_EDGED_HOMEDIR_PATH: &str = "/var/lib/aziot/edged";
+
+/// The ID used for the device ID key (symmetric or X.509 private) and the device ID cert.
+const DEVICE_ID_ID: &str = "device-id";
+
+/// The ID used for the device CA X.509 private key and the device CA cert.
+const DEVICE_CA_ID: &str = "device-ca";
+
+/// The ID used for the trust bundle certs file.
+const TRUST_BUNDLE_ID: &str = "trust-bundle";
```

Comment:
@massand Are these correct? I haven't been following your and Gordon's work but I thought I saw some mention of hard-coded IDs for these two?

Reply:
This​ PR migrates the hardcoded certid/keyid for the CA just fine - this is correct. This should work fine for the E2E tests and migrating non-quickstart scenarios (not sure if it handles the quickstart certs - in which case we should generate the `device-ca` and `trust-bundle` certs in `iotedge init` in all cases. I can add more migrate logic to my PR once you merge your PR.

Reply:
Never mind the E2E tests - we don't migrate in E2E yet. The E2E tests set the same certificates manually without `iotedge init` today and can continue to do so.

Reply:
Actually, the latest names in my PR #4088  are as follows:
```
pub const AZIOT_EDGED_CA_ALIAS: &str = "aziot-edged-device-ca";
pub const AZIOT_TRUST_BUNDLE_ALIAS: &str = "aziot-trust-bundle";
```

Could you change it to match that?

Reply:
Or if that PR doesn't go in today, we need to match the current names in the codebase - 
aziot-edged-device-ca
iotedge-trust-bundle

Reply:
Fixed. I made it use the existing constant in `edgelet_core`, and added another constant in `edgelet_core` for the trust bundle.